packages/driver/package.json

```
@@ -56,6 +56,7 @@
     "parse-domain": "bahmutov/parse-domain#fb60bd4",
     "setimmediate": "1.0.5",
     "sinon": "8.1.1",
+    "snap-shot-core": "^7.4.0",
```

Should be exact dep version, not ^
```suggestion
    "snap-shot-core": "7.4.0",
```